third-party/realdds/include/realdds/dds-stream-base.h

```diff
@@ -45,6 +45,14 @@ class dds_stream_base : public std::enable_shared_from_this< dds_stream_base >
     int default_profile_index() const { return _default_profile_index; }
     dds_options const & options() const { return _options; }
 
+    std::shared_ptr< dds_stream_profile > default_profile() const
+    {
+        std::shared_ptr< dds_stream_profile > profile;
```

Comment:
That means that there is no default profile?
I thought that we always have a default, 0 if not mentioned otherwise...

Reply:
It just means it returns null if the index isn't right... I didn't change anything with how the default is set or determined